package.json

```diff
@@ -40,8 +40,8 @@
     "build:full": "npm run clean:lerna && npm run bootstrap && npm test",
     "pretest": "npm run clean && npm run build",
     "test": "node packages/build/bin/run-nyc npm run mocha --scripts-prepend-node-path",
-    "verify:docs": "npm run build:site && node packages/build/bin/run-clean \"sandbox/loopback.io\"",
```

Comment:
I guess this was the main cause for the issue seen in https://travis-ci.org/strongloop/loopback-next/jobs/369943689?

Reply:
Yes